db/dcrsqlite/sqlite.go

```diff
@@ -651,14 +651,14 @@ func (db *DB) RetrieveBlockSummaryByTimeRange(minTime, maxTime int64, limit int)
 	return blocks, nil
 }
 
-// RetrieveDiff returns the difficulty for block ind
+// RetrieveDiff returns the difficulty for block ind at the specified chain height.
```

Comment:
I meant to replace the block ind stuff. `// RetrieveDiff returns the difficulty at the specified chain height.` If you feel it needs to directly refer to the input parameter name, it would be clearest to rename it to `height` or say `... at the specified chain height, ind.`  Personally I think it's clear enough.

Reply:
ok. Noted